environment.yml

```diff
@@ -15,6 +15,8 @@ dependencies:
   - ncurses=6.1=hfc679d8_1
   - r-base
   - r-rcpp
+  - r-xml
```

Comment:
Why do we need `r-cpp` and `r-xml` from conda? They seem to be also installed via r_requirements.txt?

Reply:
The lines have been removed from r_requirements.txt as they should be loaded through conda to be working correctly. Thanks.

Reply:
Why is that? Are the packages on CRAN broken? Or do they need additional dependencies not available through CRAN?

Reply:
Hi Bouwe,

In some cases, the compilation/installation was crashing for some packages when downloaded from CRAN. Only by installing libgfortran-ng, r-rcpp and r-xml with Conda solved the problem for us.

We tried to keep the list of dependencies in environment.yml as short as possible, but we couldn't avoid these.

If you want, you can try removing these and seeing if the installation test succeeds but, according to our experience, it should crash.

Reply:
I'm sure libgfortran-ng can be safely removed if you have the fortran compiler installed by your OS as recommended in the README.md file, this also works fine with the `version2_development` branch.

Regarding the other packages, it would be nice if could try, but if you don't have the time you can leave it for now and I will try to clean up later.